lint-tasks.yml

```diff
@@ -1,18 +1,22 @@
 version: "3"
 
 vars:
+  NODEJS_18_BIN_DIR: "{{.TASKFILE_DIR}}/build/nodejs_18/node/bin"
+  NODEJS_18_BUILD_DIR: "{{.TASKFILE_DIR}}/build/nodejs_18"
```

Comment:
We can make the above depend on the below as long as we [reorder them](https://taskfile.dev/api/#variable).